src/osx/cocoa/colour.mm

```diff
@@ -63,57 +127,65 @@ static void Draw(void *info, CGContextRef ctx)
                         CGImageRef image = (CGImageRef) info;
                         CGContextDrawImage
                         (
-                            ctx,
-                            CGRectMake(0, 0, CGImageGetWidth(image), CGImageGetHeight(image)),
-                            image
-                        );
+                         ctx,
+                         CGRectMake(0, 0, CGImageGetWidth(image), CGImageGetHeight(image)),
+                         image
+                         );
                     }
-
+                    
                     static void Release(void * WXUNUSED(info))
                     {
                         // Do not release the image here, we don't own it as it
                         // comes from NSImage.
                     }
                 };
-
+                
                 const CGPatternCallbacks callbacks =
                 {
                     /* version: */ 0,
                     &PatternCreateCallbacks::Draw,
                     &PatternCreateCallbacks::Release
                 };
-
+                
                 CGPatternRef pattern = CGPatternCreate
-                                       (
-                                            cgimage,
-                                            CGRectMake(0, 0, size.width, size.height),
-                                            CGAffineTransformMake(1, 0, 0, 1, 0, 0),
-                                            size.width,
-                                            size.height,
-                                            kCGPatternTilingConstantSpacing,
-                                            /* isColored: */ true,
-                                            &callbacks
-                                       );
+                (
```

Comment:
This whitespace-only change also looks rather unjustified to me, i.e. the old version was IMHO more readable...

Reply:
OK, I'll reindent manually